packages/components/src/unit-control/styles/unit-control-styles.ts

```diff
@@ -114,7 +115,7 @@ const unitSelectSizes = ( { selectSize = 'default' }: SelectProps ) => {
 			align-items: center;
 
 			&:where( :not( :disabled ) ):hover {
-				box-shadow: inset 0 0 0
+				box-shadow: 0 0 0
```

Comment:
I'd be curious to know the reason for using `inset` in the first place

Reply:
I think it was copypasta from the older 36px size.

<img width="279" alt="UnitControl 36px size with inset box-shadow" src="https://github.com/user-attachments/assets/e9675cd9-a0c1-4637-bf4e-0bb3fcad9399">